src/features/flows/nRF9151/Evaluate.tsx

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+import React, { useState } from 'react';
+import {
+    Button,
+    logger,
+    openUrl,
+    Spinner,
+    telemetry,
+} from '@nordicsemiconductor/pc-nrfconnect-shared';
+import describeError from '@nordicsemiconductor/pc-nrfconnect-shared/src/logging/describeError';
+
+import { useAppDispatch, useAppSelector } from '../../../app/store';
+import { Back } from '../../../common/Back';
+import Copy from '../../../common/Copy';
+import { formatResponse } from '../../../common/formatATResponse';
+import Main from '../../../common/Main';
+import { Next } from '../../../common/Next';
+import { IssueBox } from '../../../common/NoticeBox';
+import { getSelectedDeviceUnsafely } from '../../device/deviceSlice';
+import getUARTSerialPort from '../../device/getUARTSerialPort';
+import {
+    getAttestationToken,
+    getFailed,
+    setAttestationToken,
+    setFailed,
+} from './nrf9151Slice';
+
+const nRFCloudLink =
+    'https://docs.nordicsemi.com/bundle/nrf-cloud/page/GettingStarted.html';
```

Comment:
The link doesn't say how to copy the token. Is it meant to be somewhere else or the link is wrong?

Reply:
Not sure what you're asking.
We have an explanation in the step saying "Copy the token and follow the instructions".
The copying can be done with the copy icon beside the token (like in the verify step if you remember that).

Reply:
Maybe you updated the image since you opened the PR. I only now see the token on the image.

Reply:
Beside the token there is a copy icon (it's the 2 overlapping rectangles).
I did update the image but that only removed the text "Attestation token:". In the previous image neither the token nor the copy button was displayed.

Reply:
Ok.